config/rbac/auth_proxy_role.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Do these RBAC files replace similar that is in the manifest? provided by OLM?

Reply:
These RBAC replace the manual crafted RBAC files in the manifests folger namely:
![image](https://user-images.githubusercontent.com/152312/126755452-e2bc2b99-cd47-42c6-ad9a-72d0d92a9a3d.png)